Plugins/Flow.Launcher.Plugin.ProcessKiller/Main.cs

```diff
@@ -83,14 +83,14 @@ private List<Result> CreateResultsFromProcesses(List<ProcessResult> processlist,
                 results.Add(new Result()
                 {
                     IcoPath = path,
-                    Title = p.ProcessName + " - " + p.Id,
+                    Title = p.ProcessName + " - " + p.Id + (pr.Port!=0? $" - [{pr.Port}]":""),
```

Comment:
Please use string interpolation.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated

By the way, please add space around `!=` and `:`.